crates/cargo-remove/tests/testsuite/cargo_remove/mod.rs

```diff
@@ -34,13 +34,6 @@ fn init_registry() {
 
 fn add_registry_packages(alt: bool) {
     for name in [
-        "my-package",
-        "my-package1",
-        "my-package2",
-        "my-dev-package1",
-        "my-dev-package2",
-        "my-build-package1",
-        "my-build-package2",
```

Comment:
Currently these tests aren't using these fake package names, but real ones instead (albeit with fake versions). Is it worth taking the time to switch to something more generic sounding?